include/CFL/CFLGrammar.h

```diff
@@ -42,15 +42,15 @@ class GrammarBase
     typedef u32_t Kind;
     Map<std::string, Symbol> terminals;
     Map<std::string, Symbol> nonterminals;
-    Symbol startSymbol;
-    Symbol totalSymbol;
+    Symbol startKind;
+    Symbol totalKind;
     Map<Symbol, Productions> rawProductions;
-    Set<Symbol> attributeSymbol;
-    Map<Kind,  Set<Attribute>> kind2AttrMap;
+    Set<Symbol> attributeKinds;    
```

Comment:
Set<Symbol> attributeKinds;    

name is not consistent with its declaration or `Set<Kind>` ??

Reply:
Yes, that should be Set<Kind>